dpuVendor.py

```diff
@@ -63,6 +59,9 @@ def download_p4_tar(self, rh: host.Host) -> None:
         rh.run_or_die(f"curl -L {self.P4_URL} -o /tmp/p4.tar.gz")
         rh.run("rm -rf /opt/p4")
         rh.run_or_die("tar -U -C /opt/ -xzf /tmp/p4.tar.gz")
+        rh.run("mv /opt/intel-ipu-acc-components-2.0.0.11126 /opt/p4")
+        rh.run("mv /opt/p4/p4-cp /opt/p4/p4-cp-nws")
+        rh.run("mv /opt/p4/p4-sde /opt/p4/p4sde")
```

Comment:
The vsp code should be clever enough to know what to move or read different paths. Everythnig that we write here complicates the use of our solution since it will need to be mentioned in the docs. Can you patch the vsp to be more clever on what files it needs?

Reply:
You are right. Let's create a jira and mark it for now. It is a workaround because the new renamed directories don't play well. We will take it up soon but maybe after TP. You can assign it to me